src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -26,7 +28,14 @@ public synchronized void setCurrentDriver(Job2dDriver driver) {
 	 * @return Current driver.
 	 */
 	public synchronized Job2dDriver getCurrentDriver() {
-		return currentDriver;
+		if (MainFeature.loggerActive) {
+			final DriverComposite driverComposite = new DriverComposite();
+			driverComposite.add(currentDriver);
+			driverComposite.add(new LoggerDriver());
```

Comment:
This solution is tailored only for LoggerDriver. DriverManager should not be directly dependent on particular "features". Apply design that will be OCP!